SQF/dayz_code/stringtable.xml

```diff
@@ -16858,6 +16858,119 @@
+			<Spanish>No se pueden recuperar piezas en una zona segura</Spanish>
+			<Russian>Вы не можете снять детали в безопасной зоне</Russian>
+			<French>Vous ne pouvez pas récupérer des pièces dans une zone sécurisée</French>
+			<Czech>Nemůžete zachránit díly v bezpečné zóně</Czech>
+			<German>Du kannst in einer Safezone keine Teile demontieren</German>
+			<Dutch>Je kunt niet delen redden in een veilige zone</Dutch>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_BIO">
+			<English>Biology Notebook</English>
+			<German>Biologie Notizbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_BIO_DESC">
+			<English>A notebook with many notes about biological connections.</English>
+			<German>Ein Notizbuch mit vielen Notizen über biologische Zusammenhänge.</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_CHE">
+			<English>Chemistry Notebook</English>
+			<German>Chemie Notizbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_CHE_DESC">
+			<English>A notebook with many notes about chemical connections.</English>
+			<German>Ein Notizbuch mit vielen Notizen über chemische Zusammenhänge.</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_PHY">
+			<English>Physics Notebook</English>
+			<German>Physik Notizbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_PHY_DESC">
+			<English>A notebook with many notes about physical connections.</English>
+			<German>Ein Notizbuch mit vielen Notizen über physikalische Zusammenhänge.</German>
+		</Key>	
+		<Key ID="STR_EQUIP_NOTEBOOK_SECRET">
+			<English>Secret Notebook</English>
+			<German>Geheimes Notizbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_SECRET_DESC">
+			<English>In the notebook are some kind of secret information written. The text is encrypted with a secret code.</English>
+			<German>In dem Notizbuch befinden sich offenbar geheime Informationen. Der Text ist mit einen Geheimcode verschlüsselt.</German>
+		</Key>	
+		<Key ID="STR_EQUIP_NOTEBOOK_HERBS">
+			<English>Herbs Notebook</English>
+			<German>Kräuter Notizbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_HERBS_DESC">
+			<English>That notebook contains many information about herbs. It is very well documented and illustrated with pictures.</English>
+			<German>Das Notizbuch beinhaltet viele Informationen über Kräuter. Es ist sehr gut dokumentiert und illustriert.</German>
+		</Key>	
+		<Key ID="STR_EQUIP_DOCUMENT_RANDOM">
+			<English>Random Document</English>
+			<German>Irgendein Dokument</German>
+		</Key>
+		<Key ID="STR_EQUIP_DOCUMENT_RANDOM_DESC">
+			<English>Looks like a normal document, nothing special.</English>
+			<German>Sieht wie ein normales Dokument aus, nicht besonderes.</German>
+		</Key>	
+		<Key ID="STR_EQUIP_DOCUMENT_MILITARY">
+			<English>Military Document</English>
+			<German>Militärdokument</German>
+		</Key>
+		<Key ID="STR_EQUIP_DOCUMENT_MILITARY_DESC">
+			<English>A military document with many information about weapons, ammunition, tanks and helicopters.</English>
```

Comment:
`document with many information about`
should be
`document with a lot of information about`